feat: add 2019-02-07 version (#36)
ulule/developers.ulule.com: content/overview/versions.md

## Versions changelog

### 2019-02-07

> * This version changes the [location resource](#location-resource) of a project

Reviewer: can we be more precise? which fields are changing?

Author: Well, location resource was not documented but I can list the removed fields

Reviewer: please do it :+1:

> * This version changes the [location resource](#location-resource) of a project: `name`, `full_name` and `region` are removed.

Reviewer: ```suggestion
* This version removes the `full_name` , `name`, and `region` fields from the [location resource](#location-resource).
```
